docs/node/activities.md

```diff
@@ -0,0 +1,47 @@
+# Activities
+
+In Temporal, Activities are typically used to represent side effects, like making an HTTP request.
+Unlike [Workflows](/docs/node/logging#logging-from-workflows-and-activities), activities run in the standard Node.js environment, not an [isolate](https://www.npmjs.com/package/isolated-vm).
```

Comment:
Is there a better page we can link for workflows? Maybe the one about determinism?